SPECS/cloud-init/cloud-init.spec

```diff
@@ -132,21 +131,23 @@ make check %{?_smp_mflags}
 %dir %{_sysconfdir}/cloud/templates
 %doc %{_sysconfdir}/cloud/cloud.cfg.d/README
 %doc %{_sysconfdir}/cloud/clean.d/README
-%{_sysconfdir}/dhcp/dhclient-exit-hooks.d/hook-dhclient
-%{_sysconfdir}/NetworkManager/dispatcher.d/hook-network-manager
```

Comment:
Just checking, are these safe for us to remove? Is there an upstream cloud-init change associated with this removal?

Reply:
Those two files were removed in cloud-init 23.2 version and thus causing build errors. See related PR that removes those files: https://github.com/canonical/cloud-init/commit/2ee614aa6431f4339f48ebabdf2de4ebcc2401ab